src/udev/udev-builtin-net_id.c

```diff
@@ -821,13 +821,19 @@ static int names_devicetree(sd_device *dev, const char *prefix, EventMode mode)
         ofnode_path--;
         assert(path_is_absolute(ofnode_path));
 
+        /* Check if there are 'ethernet-ports' child nodes of ethernet */
+        r = sd_device_new_child(&eth_ports_dev, ofnode_dev, "ethernet-ports");
```

Comment:
Please introduce a new naming scheme flag for this. See src/shared/net-naming-scheme.[ch].

Reply:
Fixed. I've also noted I should add an entry to `man/systemd.net-naming/scheme.xml`, so I did it. Please let me know if this is not correct, thanks!